docs/guides/sqlite3_db_with_concord.md

```diff
@@ -0,0 +1,261 @@
+
+### Reading the saved records
+
+Finally, after saving the records, we wish to read these records. This is a little more complicated than creating the table and saving records. See the following example:
+
+```c
+sqlite3_stmt *stmt = NULL;
+
+query = sqlite3_mprintf("SELECT user_id FROM concord_guides WHERE guild_id = %"PRIu64";", msg->guild_id);
+rc = sqlite3_prepare_v2(db, query, -1, &stmt, NULL);
+
+if ((rc = sqlite3_step(stmt)) != SQLITE_ERROR) {
+    printf("Sucessfully read the record: %lld.\n", sqlite3_column_int64(stmt, 0));
+}
+
+if (sqlite3_finalize(stmt) != SQLITE_OK) {
+    log_fatal("[SQLITE] Error while executing function sqlite3_finalize.");
+    abort();
+}
+
+sqlite3_close(db);
+```
+
+Here, instead of the `sqlite3_exec` (literally the `sqlite3_prepare_v2` wrapper), we are using the `sqlite3_prepare_v2` for better control of it.
+
+Now, the command is "`SELECT user_id FROM user_voice WHERE guild_id = %"PRIu64";`", where it wants to get the value of user_id, using as a search parameter the guild_id also saved before.
+
+After using the `sqlite3_mprintf` to generate the query, and preparing the statement with `sqlite3_prepare_v2`, we use the `sqlite3_step` that **evaluates the statement**, and if it returns SQLITE_ROW, it is going to successfully printf the records with `sqlite3_column_int64` (NOTE: **You may wish to use other `sqlite3_column_*` to get other types of records, like TEXT**).
+
+### Deleting records
+
+After creating a table, saving records, and reading the records, you may wish to delete those records.
+
+Deleting records (AKA data) is as easy (and pretty similar) to create a table, see the following example:
+
+```c
+query = sqlite3_mprintf("DELETE FROM concord_guides WHERE guild_id = %"PRIu64";", msg->guild_id);
+rc = sqlite3_exec(db, query, NULL, NULL, &msgErr);
+
+if (rc != SQLITE_OK) {
+    log_fatal("[SYSTEM] Something went wrong while deleting concord_guides table from guild_id. [%s]", msgErr);
+    if (sqlite3_close(db) != SQLITE_OK) {
+        log_fatal("[SQLITE] Failed to close sqlite db. [%s]", sqlite3_errmsg(db));
+        abort();
+    }
+}
+
+sqlite3_close(db);
+```
+
+The only difference between the create a table and deleting records it's the command.
+
+In this command, we are **deleting the record that matches the search parameter of `guild_id = %"PRIu64"`**, which means that it will **delete the concord_guides record of the user_id** that has the guild_id the same as the guild_id of the guild that the command is being executed.
+
+After it, apply the same process as saving records. If you wish to get a better explanation of it, read `Saving records into the created table`.
+
+### Example bot with the code of all processes
+
+```c
+#include <string.h>
+
+#include <concord/discord.h>
+#include <concord/discord-internal.h>
```

Comment:
```diff
-#include <concord/discord-internal.h>
```
This should be left out! We don't want to have newcomers relying on this internal header